### ISSUE TITLE
Add darwin/arm64 support

### ISSUE THREAD

**Issue by panlilu** [valyala/gozstd#34]
MacBook with M1 chip

**Comment by valyala** [comment]
Could you check out `gozstd` source code and run `GOOS=darwin GOARCH=arm64 make libzstd.a` from the repository root on a MacBook with M1 chip? This should create `libzstd_darwin_arm64.a` file in the root folder of the repository. Then manually create `libzstd_darwin_arm64.go` file with the following contents:

```go
package gozstd

/*
#cgo LDFLAGS: ${SRCDIR}/libzstd_darwin_arm64.a
*/
import "C"
```

Then verify whether everything works by running `make test`. If all tests pass, then make a pull request with the added files `libzstd_darwin_arm64.a` and `libzstd_darwin_arm64.go`

**Comment by petersondmg** [comment]
> ```go
> package gozstd
> 
> /*
> #cgo LDFLAGS: ${SRCDIR}/libzstd_darwin_arm64.a
> */
> import "C"
> ```

Hello, I have the some problema with my M1. I followed the steps you described and all tests worked.

**Comment by valyala** [comment]
The `libzstd_darwin_arm64.a` file has been included in [v1.11.0](https://github.com/valyala/gozstd/tree/v1.11.0). Closing the issue as done.
